src/components/dashboard/ReceiveAmount.js

```diff
@@ -98,4 +103,35 @@ ReceiveAmount.shouldNavigateToComponent = props => {
   return !!screenState.nextRoutes && screenState.amount
 }
 
-export default ReceiveAmount
+const getStylesFromProps = ({ theme }) => {
+  return {
+    tableRow: {
+      // TODO: see where should we take this color from
+      borderBottomColor: '#CBCBCB',
+      borderBottomWidth: normalize(1),
+      borderBottomStyle: 'solid',
+      marginTop: theme.paddings.defaultMargin * 2,
+      alignItems: 'baseline'
+    },
+
+    // TODO: all this properties can be removed once we merge Text component in
+    tableRowLabel: {
+      color: '#A3A3A3'
+    },
+    bigGoodDollar: {
+      color: theme.colors.primary
+    },
+    name: {
+      fontSize: normalize(24),
+      fontWeight: 'bold'
+    },
+    reason: {
+      fontSize: normalize(16)
```

Comment:
`Text` component default size is 16, this shouldn't be necessary.

Reply:
Section.Text default is 14 but I'm passing this as a property anyway